jakarta/jaxrs-resource-server/README.md

```diff
@@ -20,7 +20,7 @@ To compile and run this quickstart you will need:
 
 * JDK 17
 * Apache Maven 3.8.6
-* Wildfly 28+
+* Wildfly <= 23
```

Comment:
@vaceksimon not necessary OIDC is part of WF 28 and latest, version 28 should remain.